content/docs/get-started/aws/create-project.md

```diff
@@ -33,6 +33,13 @@ $ pulumi new aws-typescript
 $ mkdir quickstart && cd quickstart
 $ pulumi new aws-python
 ```
+
+<div class="language-prologue-csharp"></div>
+
+```bash
+$ mkdir quickstart && cd quickstart
+$ pulumi new aws-csharp
```

Comment:
non-blocking: we might want to consider prioritizing azure.  but that can come later.